docs/src/contributing/guide.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+Turing is an open source project. If you feel you have some relevant skills and are interested in contributing, then please do get in touch. You can contribute by opening issues on Github or implementing things yourself and making a pull request. We would also appreciate example models written using Turing.
+
+Turing has a [style guide](../style_guide.html). It is not strictly necessary to review it, but particularly adventurous contributors may enjoy reviewing it.
```

Comment:
Could we maybe rephrase this to something like: "It's not necessary to review it before making a PR, but you may be asked to change bits of your code to conform with the style guide before it gets merged." Personal preference though really.